src/tctalker/tctalker.py

```diff
@@ -55,21 +45,10 @@ def _load_json(name):
 class TCTalker(object):
     """The base TCTalker class"""
 
-    def __init__(self, options=None):
-        o = dict()
-        o.update(_defaultConfig)
-        if options:
-            o.update(options)
-        certificate = o['credentials']['certificate']
-        try:
-            json.loads(certificate)
-        except TypeError:
-            o['credentials']['certificate'] = json.dumps(certificate)
-
-        self.options = o
-        self.queue = taskcluster.Queue(self.options)
-        self.scheduler = taskcluster.Scheduler(self.options)
-        log.debug("Dict of options: %s", self.options)
+    def __init__(self, options):
+        self.queue = taskcluster.Queue(options)
+        self.scheduler = taskcluster.Scheduler(options)
+        log.debug("Dict of options: %s", options)
```

Comment:
Are you sure it works without the escaping done via json loads/dumps operations?